crc/models/study.py

```diff
@@ -165,18 +186,14 @@ def update_model(self, study_model: StudyModel):
         if status == StudyStatus.open_for_enrollment:
             study_model.enrollment_date = self.enrollment_date
```

Comment:
Should changes to enrollment date be stored in study events as well?

Reply:
It is tracked with a timestamp, but not with the user entered date. As this change **most** of the time should be final, I opted for having the enrollment date reside in the study itself.